src/Propel/Generator/Builder/Util/DataRow.php

```diff
@@ -12,6 +12,9 @@
 
 use Propel\Generator\Model\Table;
 
+/**
+ * @deprecated Not in use
```

Comment:
Do you want to remove it instead?

This became unused when we removed `PgsqlDataSQLBuilder` in `fbf0fa08  Fix PHPStan detected issues.` Which became unusable in `f86ed34f Removed DataSQLBuilder stuff as its not in use.`.